### PR TITLE
Add implementation docs

### PR DESCRIPTION
**Pull request by maya**
This adds implementation docs for max-width text, lists, buttons, and grids.

### CODE REVIEW

**Comment by vgvg** [comment]
:+1:

**Comment by jpyuda** [comment]
This merge needs to be reverted immediately. These are not in scope for MVP/soft launch and present a serious risk to timelines. @vgvg

File: _visual/typography.md [hunk 1]
```diff
@@ -1039,11 +1043,13 @@ order: 01
...
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
+  <h4 class="usa-heading">Implementation</h4>
+  <p>Lists are styled by default. For unstyled lists, use either the <code>usa-unstyled-list</code> class or unstyled list mixin: <code>@include unstyled-list;</code>. Both are located in <code>assets/_scss/core/utilities.scss</code>.</p>
     <h4 class="usa-heading">Usability</h4>
     <h5>When to use</h5>
     <ul class="usa-content-list">
-        <li>Use an ordered list when you need to display text in some ranking, hierarchy or series of steps.</li>
-        <li>Use unordered lists display text in no specific order.</li>
+      <li>Use an ordered list when you need to display text in some ranking, hierarchy or series of steps.</li>
```
Comment (alexose) [inline _visual/typography.md]: Oxford comma... ynm?

Comment (maya) [inline _visual/typography.md]: I just removed the extra space - this was written previously. I believe 18F recommends using the Oxford comma. @awfrancisco will be proofreading tomorrow.

**Pull request merged**